backend/prisma/schema.prisma

```diff
@@ -7,28 +7,12 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-// model Post {
-//   id        Int      @id @default(autoincrement())
-//   createdAt DateTime @default(now())
-//   updatedAt DateTime @updatedAt
-//   title     String   @db.VarChar(255)
-//   content   String?
-//   published Boolean  @default(false)
-//   author    User     @relation(fields: [authorId], references: [id])
-//   authorId  Int
-// }
-
-// model Profile {
-//   id     Int     @id @default(autoincrement())
-//   bio    String?
-//   user   User    @relation(fields: [userId], references: [id])
-//   userId Int     @unique
-// }
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique
+  firstName    String 
+  lastName String 
+  passwords String 
+  birthDate DateTime 
+}
```

Comment:
We don't need to do this for this project but I do want to bring up that sometimes we might want to consider splitting up our schema files into separate folders.

As schemas get really large, we might want to move them into different folders to make it easier to find.

i.e. this schema file might want to be `backend/prisma/user/schema.prisma`

In our project, the schema probably won't be too complicated that we don't need to split it up. Just something to think about